scripts/aws/rhel-ha-aws-check.sh

```diff
@@ -426,5 +447,11 @@ for HAPKG in ${HAPKGS}; do
     fi
 done
 
+# Install awscli from pip
```

Comment:
It's a removal, not an installation. The original script has a different text here. 

Can we just copy and paste the whole original script to make sure we replace all the needed parts?

https://github.com/liangxiao1/ami-val/blob/main/ami_val/scripts/rhel-ha-aws-check.sh